### DISCUSSION TITLE
Request: Remove formatting from notifications

### DISCUSSION THREAD

**Discussion by ghost** [upptime/upptime#899] [Ideas]
**Describe the bug**
For example, while the "xx is down" notification text works fine in GitHub issues, it doesn't format properly in Slack (and I imagine some of the other alert destination types too).

**To Reproduce**
1. Setup Slack notifications
2. Choose a non-existent site to monitor
3. Watch for the alert message

**Expected behavior**
I think the formatting is fairly minimal and if it was just removed to be consistent across the board it would be an improvement, since many different chat tools do formatting differently.

**Screenshots**
![Screenshot 2023-07-12 at 10 23 23 AM](https://github.com/upptime/upptime/assets/129085294/7b2956ac-dd34-437d-af25-e61466784237)

---

I am happy to make a PR if this is an acceptable change!

**Comment by AnandChowdhary** [comment]
You're right, happy to accept a PR!
